test/common/network/configurator_test.cc

```diff
@@ -123,8 +123,9 @@ TEST_F(ConfiguratorTest, ReportNetworkUsageDisregardsCallsWithStaleConfiguration
 
 TEST_F(ConfiguratorTest, EnumerateInterfacesFiltersByFlags) {
   // Select loopback.
-  auto loopbacks = configurator_->enumerateInterfaces(AF_INET, IFF_LOOPBACK, 0);
-  EXPECT_EQ(std::get<const std::string>(loopbacks[0]).rfind("lo", 0), 0);
+  // TODO(jpsim): Re-enable test
```

Comment:
Crashing with the following backtrace:

```
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x0)
  * frame #0: 0x000000010000a545 configurator_test`std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::__is_long() const + 37
    frame #1: 0x0000000100013f5a configurator_test`std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::__get_pointer() const + 26
    frame #2: 0x0000000100013eea configurator_test`std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::data() const + 26
    frame #3: 0x000000010000880b configurator_test`std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::rfind(char const*, unsigned long) const + 27
    frame #4: 0x0000000100008615 configurator_test`Envoy::Network::ConfiguratorTest_EnumerateInterfacesFiltersByFlags_Test::TestBody() + 85
    frame #5: 0x00000001016ab068 configurator_test`void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) + 72
    frame #6: 0x0000000101687006 configurator_test`void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) + 70
    frame #7: 0x0000000101686f61 configurator_test`testing::Test::Run() + 161
    frame #8: 0x0000000101687c27 configurator_test`testing::TestInfo::Run() + 215
    frame #9: 0x0000000101688cb8 configurator_test`testing::TestSuite::Run() + 296
    frame #10: 0x0000000101693148 configurator_test`testing::internal::UnitTestImpl::RunAllTests() + 920
    frame #11: 0x00000001016af1d8 configurator_test`bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) + 72
    frame #12: 0x0000000101692c76 configurator_test`bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) + 70
    frame #13: 0x0000000101692b6a configurator_test`testing::UnitTest::Run() + 138
    frame #14: 0x00000001001bce8c configurator_test`RUN_ALL_TESTS() + 28
    frame #15: 0x00000001001bc765 configurator_test`Envoy::TestRunner::RunTests(int, char**) + 1541
    frame #16: 0x00000001001b63fb configurator_test`main + 315
    frame #17: 0x00007fff204bef3d libdyld.dylib`start + 1
    frame #18: 0x00007fff204bef3d libdyld.dylib`start + 1
```

Reply:
The crash is avoided by reverting https://github.com/envoyproxy/envoy/pull/18821